File: Python/03. Code Related to Arrays/57. Sum of Elements in an array.py
lengthOfArray = int(input("Enter the length of an Array:"))
array = []

for i in range(lengthOfArray):
    element = int(input(f"Enter {i+1} element :"))
    array.append(element)

if array:
    sum = 0

    for element in array:
        sum += element
    print("The Sum of element is",sum)

else:
    print("No elements are entered")

# Method 2
# print(sum(array))

File: Java/03. Code Related to Arrays/57. Sum of Elements in an array.java
import java.util.Scanner;

class Main {
    public static void main(String [] args) {
        Scanner userInput = new Scanner(System.in);

        System.out.print("Enter the number of elements of the Array: ");
        int size = userInput.nextInt();
        int[] numbers = new int[size];

        for (int i = 0; i < size; i++) {
            System.out.print("Element " + (i + 1) + ": ");
            numbers[i] = userInput.nextInt();
        }

        int total = 0;

        for (int i = 0; i < size; i++) {
            total += numbers[i];
        }

        // Method 2
        // for (int num : numbers) { //for-each loop
        //     total = Math.addExact(total,num);
        // }
        
        System.out.println("The Sum of element in the array is: " + total);
        userInput.close();
    }
}